tests/test_orpo_trainer.py

```diff
@@ -148,3 +148,36 @@ def test_orpo_trainer_with_lora(self, config_name):
                     # check the params have changed - ignore 0 biases
                     if param.sum() != 0:
                         self.assertFalse(torch.equal(param, new_param))
+
+    @require_liger_kernel
+    def test_orpo_trainer_with_liger(self):
+        """Test ORPO trainer with Liger loss enabled."""
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            training_args = ORPOConfig(
```

Comment:
You might consider setting beta to a larger value here. By default, beta is set to 0.1, while alpha is 1. Adjusting beta could increase the weight of the preference loss relative to the NLL loss.

Reference:
[LinkedIn Liger Kernel - fused_linear_preference.py](https://github.com/linkedin/Liger-Kernel/blob/main/src/liger_kernel/chunked_loss/fused_linear_preference.py)

Parameter Details:

alpha (float): Weight for the NLL loss (default: 1).
beta (float): Weight for the preference loss (default: 0.1).

Reply:
so the plot was with the same parameters with and without the liger_loss flag... rather than in general... so trying to figure out why there is a difference between the two settings...